docs/source/models/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -2414,6 +2416,33 @@ Here is an example:
 
   In the final version of the observers, following the noise functions, please have all data as strings. Age must be rounded down to a whole number before applying noise.
 
+**Generating WIC IDs**
+
+To generate the WIC family ID for simulated households observed in WIC,
+sort them by when the household was first observed in WIC (it does not matter how ties are broken),
+then assign them consecutive integers.
+Zero-pad these integers to 9 digits before noising.
+
+.. note::
+  The `actual process for generating WIC family IDs <https://web.archive.org/web/20230601182852/https://www.wichands.com/Content/Docs/Help/ClinicUserManual/Module1/index.html?page=family-id-number.html>`_  involves using the year in combination with
+  a consecutive integer.
+  Our approach is simpler, while still capturing that these
+  numbers are likely to be similar and easy to mistake for one another.
+  In fact, we will tend to overstate how much this is the case by not changing
+  the prefix every year.
+
+To generate the WIC client ID for simulants observed in WIC,
+sort them by when they were first observed in WIC (it does not matter how ties are broken),
+then assign them consecutive integers.
+Zero-pad these integers to 11 digits before noising.
+
+.. note::
+  The `actual process for generating WIC client IDs <https://web.archive.org/web/20230601183736/https://www.wichands.com/Content/Docs/Help/ClinicUserManual/Module1/index.html?page=client-id-number.html>`_ involves using the clinic number, local agency number, and year in combination with
+  a consecutive integer.
+  Our approach is simpler, while still capturing that these
+  numbers are likely to be similar and easy to mistake for one another.
```

Comment:
Doesn't this make it easier to link a client with the same client in a different year (maybe this isn't a use case that we care about), since in our simulation they will always have the same number, but in reality they might have a different clinic/local agency in different years?

Reply:
Client IDs stay constant over the years, so this is realistic. The year in the ID is the _first_ year that client was ever in WIC.